modules/openfast-library/src/FAST_Lin.f90

```diff
@@ -4148,6 +4695,18 @@ FUNCTION Indx_y_ED_BladeRoot_Start(y_ED, y_FAST, BladeNum) RESULT(ED_Out_Start)
    end do
 END FUNCTION Indx_y_ED_BladeRoot_Start
 !----------------------------------------------------------------------------------------------------------------------------------
+!> This routine returns the starting index for the y_ED%NacelleMotion mesh in the FAST linearization outputs.
+FUNCTION Indx_y_ED_Nacelle_Start(y_ED, y_FAST) RESULT(ED_Out_Start)
+   TYPE(FAST_OutputFileType),      INTENT(IN )  :: y_FAST           !< FAST output file data (for linearization)
+   TYPE(ED_OutputType),            INTENT(IN )  :: y_ED             !< ED outputs at t
+   INTEGER                                      :: k                !< blade number loop
+
+   INTEGER                                      :: ED_Out_Start     !< starting index of this blade mesh in ElastoDyn outputs
+
+   ED_Out_Start = Indx_y_ED_BladeRoot_Start(y_ED, y_FAST, size(y_ED%BladeRootMotion))        ! start of last blade root
+   ED_Out_Start = ED_Out_Start + y_ED%BladeRootMotion(size(y_ED%BladeRootMotion))%NNodes*18  ! N blade roots, 6 fields with 3 components per blade.
+END FUNCTION Indx_y_ED_Nacelle_Start
```

Comment:
While this works, I'd recommend using one line instead: `ED_Out_Start = Indx_y_ED_BladeRoot_Start(y_ED, y_FAST, MaxNBlades+1)` (similar to how this is done in the function `Indx_y_ED_Platform_Start`). That way, if the number of fields on the Blade Root mesh change for linearization, you only have to change code in `Indx_y_ED_BladeRoot_Start` instead of this function, too.

Reply:
Good idea! I'll make that change for the MoorDyn linearization PR (hopefully this week).